wgsl/index.bs

```diff
@@ -3615,18 +3625,18 @@ Expressions specify how values are computed.
 
 ## Type Constructor Expressions ## {#type-constructor-expr}
 
-Type constructor expressions explicitly create a value of a given [=constructible=] type.
-Type constructor expressions explicitly create a value of a given type.
+A type constructor expression explicitly creates a value of a given [=constructible=] type.
+It must match the [=syntax/constructor_expression=] grammar rule.
 
-The scalar forms are redundant, but provide symmetry with scalar [[#conversion-expr|conversion expressions]],
+The scalar forms given here are redundant, but provide symmetry with scalar [=conversion expressions=],
 and can be used to enhance readability.
 
 The vector and matrix forms construct vector and matrix values from various combinations of components and subvectors
 with matching component types.
-There are overloads for constructing vectors and matrices that specify the dimensions of the target type without having to
+There are [=overloads=] for constructing vectors and matrices that specify the dimensions of the target type without having to
 specify the component type; the component type is inferred from the constructor arguments.
 
-See also [[#zero-value-expr]] and [[#conversion-expr]].
+See also [[#zero-value-expr]] and [[#conversion-expr]] for other specialized kinds type constructor expressions.
```

Comment:
I think this might meant to be `kinds of type` than `kinds type`

Reply:
Fixed. Thanks!

Reply:
Thanks!